src/actions/party.ts

```diff
@@ -2,115 +2,79 @@ import { ActionTypes } from './types';
 import Party from '../services/party';
 import { NavigationActions } from 'react-navigation';
 
-export class PartyActions {
-    host() {
-        return (dispatch, getState) => {
-            dispatch(NavigationActions.navigate({ routeName: 'Party' }))
-            const user = getState().user;
-            Party.host(user)
-                .then(party => {
-                    dispatch(this.subscribeToParty(party))
-                    return party;
-                })
-                .then(party => dispatch(this.setPartyRecieved(party)))
-                .catch(e => console.log(e));
-        };
-    }
-
-    join(id) {
-        return (dispatch, getState) => {
-            dispatch(NavigationActions.navigate({ routeName: 'Party' }))
-            const { user } = getState();
-            Party.join({ id, user })
-                .then(party => {
-                    dispatch(this.subscribeToParty(party))
-                    return party;
-                })
-                .then(party => dispatch(this.setPartyRecieved(party)))
-                .catch(e => console.log(e));
-        };
-    }
-
-    getInfo() {
-        return dispatch => {
-            dispatch(NavigationActions.navigate({ routeName: 'JoinParty' }));
-        }
-    }
-
-    kickPlayer(id) {
-        return getState => {
-            const { user, party } = getState();
-
-            Party.kickPlayer({
-                curentPartyId: party.id,
-                kickedId: id,
-                currentUserId: user.id
-            });
-        }
+const makeActionCreator = (type: string, ...params) => {
```

Comment:
it might be, but I think we can move it up when we do the refactor in those files, let's do it despacito.